qemu/tests/cfg/boot_with_max_queues.cfg

```diff
@@ -0,0 +1,15 @@
+- boot_with_max_queues:
+    type = boot_with_max_queues
+    image_snapshot = yes
+    queues = 256
+    scp_count = 1
+    file_transfer_client = "scp"
+    filesize = 1
+    vt_ulimit_nofile = 10240
+    Linux:
+        msi_check_cmd = "lspci -vvv -s %s | grep MSI"
+    Windows:
+        no Win2008..sp2
+	cdroms += " virtio"
+        netperf_server_link_win = "netserver-2.6.0.exe"
+        netperf_client_link_win = "netperf.exe"

```

Comment:
@kimi1978wy Seems netperf is not used in this case. Should we delete it?

Reply:
@pezhang modified, thanks for reminder